### ISSUE TITLE
Version 3.0.0 fails molecule idempotency

### ISSUE THREAD

**Issue by sandal-maker** [srsp/ansible-oracle-java#23]
Hey, could you possibly fix up idempotency and release new version?

I'm using release 3.0.0
```
- src: git+https://github.com/srsp/ansible-oracle-java
  version: 3.0.0
  name: ansible-oracle-java
```

```
  vars:
    java_version: 8
    java_subversion: 201
    java_set_java_home: true
    java_download_from: oracle
    java_install_jce: false
```
molecule file
```
---
dependency:
  name: galaxy
  options:
    role-file: requirements.yml
driver:
  name: docker
lint:
  name: yamllint
platforms:
  - name: instance
    image: centos:7
provisioner:
  name: ansible
  lint:
    name: ansible-lint
  playbooks:
    converge: playbook.yml
  config_options:
    defaults:
      vault_identity_list: ~/.vault_pass.txt
scenario:
  name: default
verifier:
  name: inspec
  lint:
    name: rubocop
```
Idempotency error:-
```
--> Action: 'idempotence'
ERROR: Idempotence test failed because of the following tasks:
* [instance] => ansible-oracle-java : Generate a temporary directory for local file storage
An error occurred during the test sequence action: 'idempotence'. Cleaning up.
```

**Comment by srsp** [comment]
Thanks for pointing this out. I introduced a basic molecule test and fixed a bit of linting and the idempotency issue. Can you check if there are any other issues I missed?

**Comment by srsp** [comment]
Fixed in release 3.0.1
